pro_tes/middleware/middleware_pipeline.py

```diff
@@ -0,0 +1,173 @@
+"""Set up the middleware pipeline."""
+import importlib
+import logging
+from typing import Type
+import requests
+
+from pro_tes.exceptions import InvalidMiddleware
+from pro_tes.middleware.middleware import AbstractMiddleware
+
+logger = logging.getLogger(__name__)
+
+
+class MiddlewarePipeline:
+    """
+    Middleware Pipeline.
```

Comment:
No point in just repeating the name. Instead make the following description shorter to make it fit on one line.